Makefile

```diff
@@ -0,0 +1,7 @@
+help: ## Show this help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+install: ## Install dependencies
+	brew install pre-commit
+	brew install bufbuild/buf/buf
+	pre-commit install
```

Comment:
Could you explore using `Brewfile` instead?

An alternative to these Makefiles is to have a devcontainer instead. See SpiceDB repo for an example

Reply:
TIL Brewfile exists! But wouldn't it require `brew tap Homebrew/bundle` before? Which kind of brings me again to this file 🤔 

I find Makefile to be a pretty good standard for these things. I've never used devcontainers.

Reply:
I think `Brewfile` also supports tap.

I'm ok with makefiles, it's just not something we have adopted widely - instead we use `mage`